logging/src/commonMain/kotlin/Logger.kt

```diff
@@ -0,0 +1,31 @@
+package com.splendo.mpp.log
+
+interface Logger {
+
+    /**
+     * Writes log with provided level and tag.
+     * @param level Log level
+     * @param tag Tag of the log record. Nullable
+     * @param message Message to be written into log
+     */
+    fun log(level: LogLevel, tag: String?, message: String)
+
+    /**
+     * Writes exception log with provided level and tag.
+     * @param level Log level
+     * @param tag Tag of the log record. Nullable
+     * @param exception Exception to be written into log
+     */
+    fun log(level: LogLevel, tag: String?, exception: Throwable)
+
+    companion object {
+        fun getLoggingComponentLogLevel(logLevel: ru.pocketbyte.hydra.log.LogLevel): LogLevel {
+            return when (logLevel) {
```

Comment:
Since this method is called so frequently, it's worth considering an optimization. 

[LogLevel](
https://github.com/PocketByte/kotlin-hydra-log/blob/a4619745d43ebecb21f97c841dd13f64838c4c43/library/src/commonMain/kotlin/ru/pocketbyte/hydra/log/LogLevel.kt) has an int field for priority, you can write an extension method on LogLevel that returns the correct value from an array.

I doubt all target platforms will be able to make an equivalent compiler / runtime optimization.